Dockerfile-5.6

```diff
@@ -1,11 +1,10 @@
-FROM phusion/baseimage:0.10.0
+FROM phusion/baseimage:0.11
 
 ENV PHP_VERSION 5.6
 
 # Basic package installation
 RUN \
   apt-get update && \
-  DEBIAN_FRONTEND=noninteractive apt-get -y install python-software-properties && \
```

Comment:
Why not do `apt-get -y install python-software-properties`?
And why was this introduced in the first place?

Reply:
To quote the commit message:

> `python-software-properties` does not exist in Ubuntu 18.04 which
Phusion baseimage 0.11 is based upon. It appears we don't need
it (anymore).

The history has no track of way it was added -- my best guess is it was needed on older Ubuntu.